docs/Self-Service-MOPs/Ad-Hoc-E2E-Job.md

```diff
@@ -0,0 +1,13 @@
+* Ad-Hoc e2e test run
+
+
+Job definition: https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/jenkins/osde2e/job-templates.yaml?ref_type=heads#L67
+
+Jenkins job link: https://ci.int.devshift.net/blue/organizations/jenkins/osde2e-parameterized-job/activity
+
+
+Configurable parameters:
```

Comment:
How do we feel about removing this?

As the job evolves, build parameters may change (added, removed, modified). Which requires us making a change to the JJB job template along with updating this document. Now that the job has [descriptions](https://ci.int.devshift.net/job/osde2e-parameterized-job/build?delay=0sec) for each build parameter and some of the ones that are required are defaults.